LICENSE.md

```diff
@@ -0,0 +1,23 @@
+# The MIT License (MIT)
+
+### Copyright © 2016 elementary & [The Web Team](https://github.com/elementary/houston/graphs/contributors)
```

Comment:
"elementary" should probably be "elementary LLC". I'm unsure about the other part. I don't know if those things hold up in court of if there's a way to do non-legal entities like that